evalml/tests/automl_tests/test_automl.py

```diff
@@ -2177,3 +2177,16 @@ def test_automl_pipeline_params_kwargs(mock_fit, mock_score, X_y_multi):
         if 'Decision Tree Classifier' in row['parameters']:
             assert 0.1 < row['parameters']['Decision Tree Classifier']['ccp_alpha'] < 0.5
             assert row['parameters']['Decision Tree Classifier']['max_depth'] == 1
+
+
+@pytest.mark.parametrize("random_state", [0, 1, 9])
+@patch('evalml.pipelines.MulticlassClassificationPipeline.score')
+@patch('evalml.pipelines.MulticlassClassificationPipeline.fit')
+def test_automl_pipeline_random_state(mock_fit, mock_score, random_state, X_y_multi):
+    X, y = X_y_multi
+    automl = AutoMLSearch(X_train=X, y_train=y, problem_type='multiclass', random_state=random_state, n_jobs=1)
+    automl.search()
+
+    for i, row in automl.rankings.iterrows():
+        if 'Base' not in list(row['parameters'].keys())[0]:
```

Comment:
O same question as other PR LOL, why the check for `Base` here :o

Reply:
The baseline pipeline isn't initialized with a `random_state` [arg](https://github.com/alteryx/evalml/blob/4b3bfe4d000da71973d4044ab39623bb478723c5/evalml/automl/automl_search.py#L635)